engine/glfw/lib/android/android_init.c

```diff
@@ -211,9 +218,13 @@ static void handleCommand(struct android_app* app, int32_t cmd) {
             // application will attempt to open the GL window before it has regained focus.
             g_appLaunchInterrupted = 1;
         }
-        destroy_gl_surface(&_glfwWin);
-        _glfwWin.hasSurface = 0;
+
+        spinlock_lock(&_glfwWinAndroid.m_RenderLock);
+
+        destroy_gl_surface(&_glfwWinAndroid);
```

Comment:
We need to destroy the surface while the window exists.
(we cannot delay it to the engine thread)